### ISSUE TITLE
When selecting specific partition days it is only processing as per the incremental refreshing policy

### ISSUE THREAD

**Issue by GilbertQue** [m-kovalsky/ProcessingManager#4]
Hi there, thanks for fixing the previous issue :)

What I now did was to configure the Partitions as shown below
![image](https://user-images.githubusercontent.com/72849240/135533977-b009ce84-47a7-4831-9a00-7cad8c2f1c78.png)

Now when I run the command line prompt, I watch it via SQL Profiler and I see it is not processing the partitions I selected.
![image](https://user-images.githubusercontent.com/72849240/135534070-8487ed58-618e-4458-b10a-9cc25aa7f2c2.png)

And when I review the partitions processed date I can see that the above partitions have not been processed.
![image](https://user-images.githubusercontent.com/72849240/135534180-70613e69-16a2-4430-9f3c-a158ecad6f4c.png)

If you need anything further for me to help please let me know.

**Comment by m-kovalsky** [comment]
This is due to the Incremental Refresh policy. This tool is not set up to override an Incremental Refresh policy. Would you share the details of the IR policy?

**Comment by GilbertQue** [comment]
Hi Michael, thanks for getting back to me.

Here are the details of the IR policy

![image](https://user-images.githubusercontent.com/72849240/135933467-43b40438-6529-4925-a14d-bfe05f0f8920.png)

If you need any more details please let me know.

**Comment by m-kovalsky** [comment]
If you want it to ignore the Incremental Refresh policy, I believe you should set the table(s) as such:

![image](https://user-images.githubusercontent.com/29556918/135982150-9e1e8c4a-190c-4dfc-9197-4e12e809ada9.png)

**Comment by GilbertQue** [comment]
Thanks Michael, just to confirm the steps should be.

1. Disable the "EnableRefreshPolicy" 
2. Run the processing of the partitions with your amazing script.
3. Re-Enable the "EnableRefreshPolicy"

**Comment by m-kovalsky** [comment]
Yes, try it out and let me know. You can also run steps 1&3 as scripts. If this works, I can add a functionality within the tool to allow an option for disabling incremental refresh just for a particular processing batch.

Step 1:
```c#
Model.Tables["TableName"].EnableRefreshPolicy = false;
```
Step 3
```c#
Model.Tables["TableName"].EnableRefreshPolicy = true;
```

**Comment by GilbertQue** [comment]
Hi Michael,

Thanks for the suggestion when I try and do this via Tabular editor to disable the refresh policy on a PPU database I get the following error shown below

![image](https://user-images.githubusercontent.com/72849240/136503787-d9e063c9-9f71-4317-99ba-e40090c1ff85.png)

**Comment by m-kovalsky** [comment]
This issue is a bit outside the scope of this tool. The error message isn't too helpful but make sure to check out Tabular Editor's documentation on the subject. Otherwise you may want to open a support ticket with Microsoft.

https://docs.tabulareditor.com/te2/incremental-refresh.html
